monai/config/check_env.py

```
@@ -0,0 +1,181 @@
+
+    dcount = torch.cuda.device_count()
+    fprint("CUDA version:", torch.version.cuda)
+    fprint("PyTorch GPU Count:", dcount)
+
+    try:
+        for d in range(dcount):
+            fprint(f"  {torch.cuda.get_device_properties(d)}")
+            t = torch.rand(2, 3).to(torch.device(f"cuda:{d}")) * 5
+            fprint("Test tensor:", t.flatten())
+        return True
+    except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)